numpy/lib/function_base.py

```diff
@@ -1492,8 +1495,8 @@ def unwrap(p, discont=None, axis=-1, *, min_val=-pi, max_val=pi):
 
     Notes
     -----
-    If the discontinuity in `p` is smaller than ``pi``, but larger than
-    `discont`, no unwrapping is done because taking the 2*pi complement
+    If the discontinuity in `p` is smaller than ``max_val``, but larger than
```

Comment:
Are you sure this isn't (min + max)/2?

Reply:
yes that makes more sense, but anyways this should have a test that I'll add to verify

Reply:
wait I think `max_val-min_val/2` no? (min+max)/2 for the default case would be 0